libnl-route-3.sym

```diff
@@ -286,6 +286,7 @@ global:
 	rtnl_link_get_broadcast;
 	rtnl_link_get_by_name;
 	rtnl_link_get_carrier;
+	rtnl_link_get_carrier_changes;
```

Comment:
a section in the linker-version script must never be modified once a new API is released. That means, new symbols must always be added to the section for the next upcoming release, that is `libnl_3_2_29` at the moment. I mean, new symbols must be added at last section in this file.